src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,54 +1,37 @@
 package frc.robot.subsystems;
 
-import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkFlex;
```

Comment:
Why is this being done? is not documented in the commits. Please add to the PR description